app/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "alacritty"
+version = "0.3.0"
+authors = ["Joe Wilm <joe@jwilm.com>"]
+license = "Apache-2.0"
+description = "GPU-accelerated terminal emulator"
+readme = "../README.md"
+homepage = "https://github.com/jwilm/alacritty"
+edition = "2018"
+
+[dependencies]
+alacritty-core = { path = ".." }
```

Comment:
Currently this makes the root of the project the core alacritty library. However I feel like organization becomes kinda cluttered at that point. I think moving core to `alacritty-core` directory so the root doesn't have any source directory would make things a bit more uniform.

Similar to what [glutin](https://github.com/tomaka/glutin) and many others do already.